core/src/test/java/io/questdb/test/ServerMainBackupDatabaseTest.java

```diff
@@ -1,374 +0,0 @@
-/*******************************************************************************
- *     ___                  _   ____  ____
- *    / _ \ _   _  ___  ___| |_|  _ \| __ )
- *   | | | | | | |/ _ \/ __| __| | | |  _ \
- *   | |_| | |_| |  __/\__ \ |_| |_| | |_) |
- *    \__\_\\__,_|\___||___/\__|____/|____/
- *
- *  Copyright (c) 2014-2019 Appsicle
- *  Copyright (c) 2019-2024 QuestDB
- *
- *  Licensed under the Apache License, Version 2.0 (the "License");
- *  you may not use this file except in compliance with the License.
- *  You may obtain a copy of the License at
- *
- *  http://www.apache.org/licenses/LICENSE-2.0
- *
- *  Unless required by applicable law or agreed to in writing, software
- *  distributed under the License is distributed on an "AS IS" BASIS,
- *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- *  See the License for the specific language governing permissions and
- *  limitations under the License.
- *
- ******************************************************************************/
-
-package io.questdb.test;
-
-import io.questdb.Bootstrap;
-import io.questdb.PropertyKey;
-import io.questdb.ServerMain;
-import io.questdb.cairo.CairoEngine;
-import io.questdb.cairo.CursorPrinter;
-import io.questdb.cairo.PartitionBy;
-import io.questdb.cairo.TableToken;
-import io.questdb.cairo.sql.RecordCursor;
-import io.questdb.cairo.sql.RecordCursorFactory;
-import io.questdb.cairo.sql.RecordMetadata;
-import io.questdb.cairo.wal.ApplyWal2TableJob;
-import io.questdb.griffin.CompiledQuery;
-import io.questdb.griffin.SqlCompiler;
-import io.questdb.griffin.SqlExecutionContext;
-import io.questdb.log.LogFactory;
-import io.questdb.mp.SOCountDownLatch;
-import io.questdb.std.*;
-import io.questdb.std.str.Path;
-import io.questdb.std.str.StringSink;
-import io.questdb.test.tools.TestUtils;
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.BeforeClass;
-import org.junit.Test;
-
-import java.io.File;
-import java.util.Comparator;
-import java.util.Objects;
-import java.util.UUID;
-import java.util.concurrent.Callable;
-import java.util.concurrent.CyclicBarrier;
-import java.util.concurrent.ThreadLocalRandom;
-import java.util.concurrent.TimeUnit;
-import java.util.concurrent.atomic.AtomicBoolean;
-import java.util.concurrent.atomic.AtomicLong;
-import java.util.concurrent.atomic.AtomicReference;
-
-import static io.questdb.test.griffin.TableBackupTest.executeCreateTableStmt;
-import static io.questdb.test.griffin.TableBackupTest.executeInsertGeneratorStmt;
-import static io.questdb.test.tools.TestUtils.assertMemoryLeak;
-import static io.questdb.test.tools.TestUtils.createSqlExecutionCtx;
-
-public class ServerMainBackupDatabaseTest extends AbstractBootstrapTest {
```

Comment:
BACKUP DATABASE is the only way to do a backup on Windows as snapshots/checkpoint aren't supported on this OS. So, we should probably keep it and fix the test. WDYT?

Reply:
The backup is consistently confusing subject for our users. We decided a while ago to remove all the branches and only document snapshot (checkpoint - https://github.com/questdb/questdb/pull/4821).

Windows is not a serious production system either, the way to backup database is to stop the instance and backup.

We still have `BACKUP DATABASE`, i just don't think it is worthwhile exercise to make sure it works on ZFS (linux) :(

Reply:
The test is flaky on ext4, not only zfs. If we remove the test, backup may get broken unnoticeably. To me, as long as we keep the feature around, we should fix the test instead of removing it.

Reply:
the issues is - the backup is probably broken already. I don't want to spend time and effort fixing it though and also waste time re-running CI. "backup" SQL has already been removed from the documentation.

Reply:
Got it, let's remove the test then.

> I would remove "backup" SQL entirely, but it is just a bigger change for this PR :)

Sounds like a great idea for another PR.